VirtoCommerce.CatalogModule.Web/Scripts/blades/common/catalog-items-select.js

```diff
@@ -27,6 +27,7 @@
                     categoryId: blade.categoryId,
                     keyword: filter.keyword,
                     responseGroup: blade.responseGroup || 'withCategories, withProducts',
+                    searchInVariations: true,
```

Comment:
Can we do it optionally, to be able to control this behavior where this control uses?